claim.go

```diff
@@ -324,6 +335,18 @@ func (c *Claim) GetSchemaHash() SchemaHash {
 	return schemaHash
 }
 
+// GetSubjectPosition return a position where the subject is stored.
+func (c *Claim) GetSubjectPosition() (SubjectPosition, error) {
+	switch c.getSubject() {
+	case SubjectOtherIdenIndex:
+		return SubjectPositionIndex, nil
+	case SubjectOtherIdenValue:
+		return SubjectPositionValue, nil
+	default:
+		return "", errors.New("claim without ")
```

Comment:
If you won't agree with my previous comment, then it looks like this error needs more text.